src/service/logger/logger-service-factory.ts

```diff
@@ -4,22 +4,24 @@ import { EntryPoint } from "@bc/domain/entry-point";
 import { CLILoggerService } from "@bc/service/logger/cli-logger-service";
 import { GithubActionLoggerService } from "@bc/service/logger/github-action-logger-service";
 import { Container } from "typedi";
+import { constants } from "@bc/domain/constants";
 
 export class LoggerServiceFactory {
 
   private static instance?: LoggerService;
 
   public static getInstance(): LoggerService {
     if (!LoggerServiceFactory.instance) {
-      switch (Container.get(ConfigurationService).configuration?.entryPoint) {
+      const entryPoint: EntryPoint = Container.get(constants.CONTAINER.ENTRY_POINT);
```

Comment:
and I guess this is a very good solution where we force runners (thanks to abstract constructor expecting entryPoint) to define the entry point and you always has the entryPoint accessible (and then the logger) since the very first line.
Additionally can just `Container.set(constants.CONTAINER.ENTRY_POINT, EntryPoint.CLI);` in case the want to log or mock something